tests/src/Functional/PersonContentRenderTest.php

```diff
@@ -0,0 +1,287 @@
+      'This field is used to add a short biography of the person.',
+      $content_banner->filter('p')->text()
+    );
+
+    $this->assertEquals(
+      'Twitter profile',
+      $content_banner->filter('div.mb-2 > div.mb-3 > a.standalone')->text()
+    );
+
+    $content = $crawler->filter('div.col-12.col-lg-10.col-xl-9.col-xxl-8');
+
+    $this->assertEquals(
+      'Additional information',
+      $content->filter('h2')->eq(0)->text()
+    );
+    $this->assertEquals(
+      'Additional information example field.',
+      $content->filter('div > p')->text()
+    );
+
+    $this->assertEquals(
+      'Related documents',
+      $content->filter('h2')->eq(1)->text()
+    );
+    $this->assertEquals(
+      'Event document test',
+      $content->filter('.bcl-file p')->text()
+    );
+  }
+
+  /**
+   * Tests the person rendered in 'Teaser' view mode.
+   */
+  public function testPersonRenderingTeaser(): void {
+    $node = $this->createExamplePersonWithAllFields();
+    // Build node teaser view.
+    $builder = \Drupal::entityTypeManager()->getViewBuilder('node');
+    $build = $builder->view($node, 'teaser');
+    $markup = $this->container->get('renderer')->renderRoot($build);
+    $crawler = new Crawler((string) $markup);
+
+    $article = $crawler->filter('article');
+    $this->assertCount(1, $article);
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $article->filter('h1.card-title')->text()
+    );
+    $image = $article->filter('img.card-img-top');
+    $this->assertCount(1, $image);
+    $this->assertStringContainsString(
+      'image-test.png',
+      $image->attr('src')
+    );
+    $this->assertEquals(
+      'DG Test',
+      $article->filter('.card-body > div.my-3 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $article->filter('.card-body > div.my-3 > span:nth-child(2)')->text()
+    );
+  }
+
+  /**
+   * Creates an example person node only with required fields.
+   *
+   * @return \Drupal\node\NodeInterface
+   *   Person node.
+   */
+  protected function createExamplePersonWithRequiredFieldsOnly(): NodeInterface {
+    // Create a Person node with required fields.
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->create([
+        'type' => 'oe_sc_person',
+        'oe_sc_person_first_name' => 'Stefan',
+        'oe_sc_person_last_name' => 'Mayer',
+        'oe_sc_person_country' => 'DE',
+        'oe_sc_person_occupation' => 'DG Test',
+        'oe_sc_person_position' => 'Director',
+        'uid' => 1,
+        'status' => 1,
+      ]);
+    $node->save();
+    return $node;
+  }
+
+  /**
+   * Creates an example person node with required and optional fields.
+   *
+   * @return \Drupal\node\NodeInterface
+   *   Person node.
+   */
+  protected function createExamplePersonWithAllFields(): NodeInterface {
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = \Drupal::entityTypeManager()
```

Comment:
Don't change things between reviews without explaining why (maybe with separate commit).
I see "improve node test creation", but for me you duplicated the code, there's no improvement. Restore the call to createExamplePersonWithRequiredFieldsOnly().